enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,720 @@
+A future mitigation would be extending to support [non-PromQL filters](#non-promql-filters).
+
+## Design Details
+
+### Test Plan
+
+[The graph-data repository][graph-data] should grow a presubmit test to enforce as much of the new schema as is practical.
+Validating any PromQL beyond "it's a string" is probably more trouble than its worth, but we should keep an eye on Telemetry to see if any deployed cluster-version operators are reporting trouble with condition evaluation.
+
+Extending existing mocks and stage testing with data using the new schema should be sufficient for [update service support](#update-service-support-for-the-enhanced-schema).
+
+Adding unit tests with data from a mock Cincinnati update service should be sufficient for [cluster-version operator support](#cluster-version-operator-support-for-the-enhanced-schema).
+
+Ad-hoc testing when landing new features should be sufficient for `oc` and the web-console, although if they have existing frameworks for comparing output with mock cluster resources, that would be great too.
+
+### Graduation Criteria
+
+This will be released directly to GA.
+
+#### Dev Preview -> Tech Preview
+
+This will be released directly to GA.
+
+#### Tech Preview -> GA
+
+This will be released directly to GA.
+
+#### Removing a deprecated feature
+
+This enhancement does not remove or deprecate any features.
+
+### Upgrade / Downgrade Strategy
+
+The graph-data schema is already versioned.
+
+We have [an open RFE][ota-123] to version the Cincinnati API, but even without that, adding new optional properties (`conditionalEdges`) for new features (edges which would have previously been completely blocked) is a backwards-compatible change.
+
+### Version Skew Strategy
+
+Newer update services consuming older graph-data will know that they can use their 1.1.0 parser on 1.0.0 graph-data without needing to make changes.
+
+Older update services consuming newer graph-data will know that they are missing some features unique to 1.1.0, but that they will still get something reasonable out of the data by using their 1.0.0 parser (they'll just consider all conditional edges to be complete blockers).
+
+Newer clients talking to older update services will not receive any `conditionalEdges`, but they will understand all the data that the update service sends to them.
+Older clients talking to newer update services will not notice `conditionalEdges`, so those edges will continue to be unconditionally blocked for those clients.
+
+Newer clients consuming older ClusterVersion will not receive any `conditionalUpdates`, but they will understand all the data included in the ClusterVersion object (e.g. `availableUpdates`).
+Older clients consuming newer ClusterVersion will not notice `conditionalUpdaets`, so those edges will continue to be unconditionally blocked for those clients.
+
+## Implementation History
+
+Major milestones in the life cycle of a proposal should be tracked in `Implementation
+History.
```

Comment:
nit: There's a missing quote here that throws of the markdown parser.

Reply:
Thanks; fixed in de88e18 -> 43c3354.